subprojects/docs/docs.gradle

```diff
@@ -186,16 +175,10 @@ tasks.withType(AssembleDslDocTask) {
 
 task configureCss {
     doLast {
-        def images = fileTree(dir: "src/docs/css/images", include: "*.*").files.collectEntries {
+        ext.tokens = fileTree(dir: "src/docs/css/images", include: "*.*").files.collectEntries {
```

Comment:
I think the use of `ext` doesn't make too much sense here. I don't think access the specific action of the task from somewhere else to get ahold of field. In this case it's probably fine to have a free-standing variable. Do we reference `tokens` from somewhere else?